packages/xo-web/src/common/xoa-updater.js

```diff
@@ -41,7 +41,7 @@ export function exposeTrial (trial) {
 
 export function blockXoaAccess (xoaState) {
   let block = xoaState.state === 'untrustedTrial'
-  if (process.env.XOA_PLAN > 1 && process.env.XOA_PLAN < 5) {
+  if (process.env.XOA_PLAN <= 1 || process.env.XOA_PLAN >= 5) {
```

Comment:
Does this include connection issues with xoa-updater as well?

Reply:
Do you mean connection to xoa-updater from xo-web, or connection to www-xo from xoa-updater?

Reply:
`xoa-updater` - `xo-web`.

Reply:
I don't recognize what was implemented in the angular version I coded long ago. There was a time when the paid appliances where blocked if there was no updater answering at all (service down), this choice may have change, but anyway, with a quick reading of what is wrote today, I think the error handling of the xoa/update panel is not compliant with the "states" that the xoa-updater client module provides.

I will check all this with @pdonias ASAP.